migrations/2_pool_master.js

```diff
@@ -0,0 +1,16 @@
+const KyberPoolMaster = artifacts.require('KyberPoolMaster');
+
+const KYBER_DAO_ADDRESS = process.env.KYBER_DAO_ADDRESS;
+const EPOCH_NOTICE = process.env.EPOCH_NOTICE;
+const INITIAL_DELEGATION_FEE = process.env.INITIAL_DELEGATION_FEE;
+
+module.exports = async function (deployer) {
```

Comment:
As part of the deployment steps, right now it deploys the `Migrations` contract and then the `KyberPoolMaster`. However, there is no value in deploying the `Migration` contract for us and it consumes gas and time. I think it would be better to remove the migration `1_initial_migration.js` and rename this one to `1_pool_master.js`. Unfortunately, the `Migrations.sol` can't be deleted because truffle needs it to work.